File: C/BEGINNER/1040 Average 3.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float n1, n2, n3, n4, average;
    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
    average = ((n1 * 2 + n2 * 3 + n3 * 4 + n4 * 1) / 10);
    printf("Media: %.1f\n", average);
    if (average >= 7.0)
        printf("Aluno aprovado.\n");
    else if (average < 7.0 && average >= 5.0)
    {
        printf("Aluno em exame.\n");
        float n5;
        scanf("%f", &n5);
        printf("Nota do exame: %.1f\n", n5);
        if (((average + n5) / 2) >= 5.0)
            printf("Aluno aprovado.\n");
        else
            printf("Aluno reprovado.\n");
        printf("Media final: %.1f\n", ((average + n5) / 2));
    }
    else
        printf("Aluno reprovado.\n");
    return 0;
}

File: Python/BEGINNER/1040 Average 3.py
n1, n2, n3, n4 = map(float, input().split())

total_average = ((n1 * 2) + (n2 * 3) + (n3 * 4) + (n4 * 1)) / 10

if total_average >= 7.0:
    print('Media: %.1f' % total_average)
    print('Aluno aprovado.')
elif total_average < 5.0:
    print('Media: %.1f' % total_average)
    print('Aluno reprovado.')
else:
    print("Media: %.1f" % total_average)
    print('Aluno em exame.')

    new_score = float(input())
    print('Nota do exame: %.1f' % new_score)
    final_average = (new_score + total_average) / 2

    if final_average >= 5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print('Media final: %.1f' % final_average)
